kitsune/wiki/tests/test_templates.py

```diff
@@ -200,12 +200,24 @@ def test_document_fallback_no_translation(self):
         eq_(r.document.title, doc('article h1.title').text())
 
         # Fallback message is shown.
-        eq_(1, len(doc('#doc-pending-fallback')))
+        eq_(None, len(doc('#doc-pending-fallback')))
         # Removing this as it shows up in text(), and we don't want to depend
         # on its localization.
         doc('#doc-pending-fallback').remove()
         # Included content is English.
         eq_(pq(r.document.html)('div').text(), doc('#doc-content div').text())
+        
+        r.is_ready_for_localization=True
+        r.save()
+        
+        response = self.client.get(url)
+        doc = pq(response.content)
+        
+        eq_(r.document.title, doc('article h1.title').text())
+        # Fallback message is not shown.
+        eq_(1, len(doc('#doc-pending-fallback')))
+        # Included content is English.
+        eq_(pq(r.document.html)('div').text(), doc('#doc-content div').text())
```

Comment:
include the `doc('#doc-pending-fallback').remove()` here because its needed here